modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -4,36 +4,52 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation, jre
 
-Recommended minimum hardware is:
+If you plan to install Mule and run it on premises, review these minimum hardware and software requirements before you install.
 
-* 2GHz CPU or 1 virtual CPU in virtualized environments
-* 1GB of RAM
-* 4GB of storage
+== Minimum Hardware Requirements
+
+* 2GHz CPU or 1 virtual CPU in virtualized environments.
+* 1GB of RAM.
+* 4GB of storage.
 
 Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Required Software
+
+Verify that you use a supported version of Java before you install Mule.
+
+[%header,cols="20a,75a"]
+|===
+|Software |Version
+|JDK | JDK 1.8.0 (1.8.0_202 recommended), IBM JDK 1.8, OpenJDK 8 (latest update recommended)
+|===
+
+== Supported Software
+
+[NOTE]
+Mule runs on any operating system that runs a supported Java Runtime Environments. Mule runtime is tested against the following software and operating system resulting in a valid and correct functionality.
+Other software and operating system might work with Mule but have not been tested. It is recommended to run Mule on software and operating system versions that have been validated.
+
+[%header,cols="20a,75a"]
+|===
+|Software |Version
+|OS |MacOS 10.11.4, HP-UX 11i V3, AIX 7.2, Windows 2012 R2 Server, Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 15.04 and 16.04
+|Application servers |Tomcat 7 and 8, WebLogic 12c, WebSphere 8, WildFly 8 and 9, Jetty 8 and 9
+|Databases |Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
+|Browsers for working on Anypoint Platform | Firefox or Chrome (latest), Internet Explorer 10 or later; minimum screen resolution of 1024x768
+|===
+
+The unified Mule Runtime 3.9 is compatible with APIkit 3.9.0 and Anypoint Studio 6.4.0
 
-* One of the following Java versions:
-+
-** Oracle JDK 1.8.0 (latest update recommended)
-** IBM JDK 1.8
-** OpenJDK 8 (latest update recommended)
-+
-* For Microsoft Dynamics GP connector install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
+Mule Runtime 3.9 is bundled with the Runtime Manager Agent plugin version 1.9.0.
 
-Mule Runtime is tested against operating systems listed in the "Supported Software and Systems" table.
+For Microsoft Dynamics GP connector install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
```

Comment:
add article "the" and comma: "For the Microsoft Dynamics GP connector, install ..."

Reply:
Updated.